### ISSUE TITLE
Failed to install with latest repo

### ISSUE THREAD

**Issue by Zeusyf** [afiniel/yiimp_install_script#32]
----------------------------------------------------------> 	  		
  | Yiimp Installer Script Fork By Afiniel!					
  | Version: v0.9.2 								
   ----------------------------------------------------------------------> 	  			
  | This script will install all the dependencies and will install Yiimp.	
  | It will also install a MySQL database and a Web server.			
  | MariaDB is used for the database.						
  | Nginx is used for the Web server, PHP 7.3 is also installed.		
   ----------------------------------------------------------------------> 	  			

    <-------------------------->
     <-- System Configuration -->
    <-------------------------->

 =>  Setting TimeZone to: UTC <= 

      
 Installing MariaDB...
 Complete...
      
 => Installing Base system packages <= 
 => Complete

 => Initializing system random number generator <= 
 => Complete

 => Initializing UFW Firewall <= 
      
 => Allow incoming connections to SSH <= 

 ssh port: OPEN 

 http port: OPEN 

 https port: OPEN 

      
 =>  Installing YiiMP Required system packages <= 
      
 => Installing php7.3 
      
 =>  Clone Yiimp Repo <= 
system.sh: line 243: YiiMPRepo: unbound variable

**Comment by kushalg-1212** [comment]
I have the same error , were you able to fix it  ?

**Comment by MD-0411** [comment]
=> Clone Yiimp Repo <=
system.sh: line 243: YiiMPRepo: unbound variable
Ubuntu20.04 I also encountered the same mistake, hope to fix it, thank you!

**Comment by sp1k36667** [comment]
have it too 

System:
ubuntu 20.04
8GB Ram

**Comment by Zeusyf** [comment]
waiting answer @afiniel

**Comment by lidonia** [comment]
I went through this script, and this issue is the first of many issues in this script.  The reason line 243 fails is that it is looking for a variable that doesn't exist.  The YiimPRepo variable should point to the Github repository where yiimp is stored.  I found multiple instances where the variables are missing.  I assume that .yiimpool.conf is where the variables are stored and referenced during setup.  @afiniel should be able to shed some light on this.
